test/bigtest/tests/package-content-test.js

```diff
@@ -0,0 +1,110 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { faker } from '@bigtest/mirage';
+
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+import PackageContentInteractor from '../interactors/package-content';
+
+const PKG = {
+  name: 'access_start_access_end_tests Package'
+};
+
+const testData = [
+  { pti: { titleInstance: { id: '1234', name: 'testInstance1' } }, accessStart: () => faker.date.recent().toISOString() },
+  { pti: { titleInstance: { id: '234', name: 'testInstance2' } }, accessStart: () => faker.date.future().toISOString() },
+  { pti: { titleInstance: { id: '734', name: 'testInstance3' } }, accessEnd: () => faker.date.recent().toISOString() },
+];
+
+describe('PackageContentCRUD', () => {
+  setupApplication();
+  const packagecontent = new PackageContentInteractor();
+  let eresource;
+
+  beforeEach(async function () {
+    eresource = this.server.create('eresource', { name: PKG.name, testData });
+  });
+
+  describe('packagecontent tests', () => {
+    beforeEach(async function () {
+      this.visit(`/erm/eresources/${eresource.id}`);
+    });
+
+    describe('visiting the eresource pane', () => {
+      it('should render expected package', () => {
+        expect(packagecontent.headline).to.equal(PKG.name);
+      });
+
+      it('should render the expected current eresource', () => {
+        expect(packagecontent.eresourceName).to.equal(testData[0].pti.titleInstance.name);
```

Comment:
Avoid magic numbers like these indices into `testData` you use throughout. It makes it hard to read the first time and it'll be unimaginably harder for someone to parse through a few years from now.

We could write these sections this instead as:
```
const droppedResource = ...
const currentResource = ...
const futureResource = ...
...
package = this.server.create('eresource', { name: packageName, pcis: [ droppedResource, currentResource, futureResource ] });
...
expect(packageContent.eresourceName).to.equal(futureResource.pti.titleInstance.name)
```